config/lava/chromeos/chromeos-tast-template.jinja2

```diff
@@ -0,0 +1,46 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block actions %}
+context: 
+  extra_kernel_args: cros_debug cros_secure root=/dev/mmcblk0p3
```

Comment:
hard-code here, I will work as you suggest in a follow up improvement.

Reply:
I meant, using the `context.update()` method on the template variable so that it gets added to any other context.

See this for example:
```
config/lava/boot/generic-depthcharge-tftp-ramdisk-boot-template.jinja2:{% do context.update({"extra_kernel_args": extra_kernel_args}) %}
```

Which is by default set here:
```
config/lava/base/kernel-ci-base.jinja2:{% set extra_kernel_args = "console_msg_format=syslog earlycon" %}
```

So in this template, you could define this variable:
```
{% set extra_kernel_args = "console_msg_format=syslog earlycon" cros_debug cros_secure root=/dev/mmcblk0p3 %}
```